topics/fair/tutorials/ro-crate-galaxy-best-practices/tutorial.md

```diff
@@ -27,21 +27,45 @@ notebook:
+One of the most useful additions to a workflow is a suite of tests, which help check that the workflow is operating as intended. A test case consists of a set of inputs and corresponding expected outputs, together with a procedure for comparing the workflow's actual outputs with the expected ones. It might be the case, in fact, that a test may be considered successful even if the actual outputs do not match the expected ones exactly, for instance because the computation involves a certain degree of randomness, or the output includes timestamps or randomly generated identifiers.
+
+Providing documentation is also important to help understand the workflow's purpose and mode of operation, its requirements, the effect of its parameters, etc. Even a single, well structured README file can go a long way towards getting users started with your workflow, especially if complemented by examples that include sample inputs and running instructions.
 
 > <agenda-title></agenda-title>
 >
-> In this tutorial, you will learn how to create a git repo, and begin working with it.
+> In this tutorial, you will learn about the best practices that the Galaxy community 
+has created for workflows.
 >
 > 1. TOC
 > {:toc}
 >
 {: .agenda}
 
+> <tip-title>Using your own workflow</tip-title>
+>
+> This tutorial assumes that you already have a Galaxy workflow that you want to apply best practices to. You can follow along using any workflow you have created or imported during a previous tutorial (such as [A short introduction to Galaxy]({% link topics/introduction/tutorials/galaxy-intro-short/workflows/index.md %})).
+>
+{: .tip}
 
 ## Community best practices
 
-Though the practices listed above can be considered general enough to be applicable to any kind of software, individual communities usually add their own specific sets of rules and conventions that help users quickly find their way around software projects, understand them more easily and reuse them more effectively. The Galaxy community, for instance, has a [guide on best practices for maintaining workflows](https://planemo.readthedocs.io/en/latest/best_practices_workflows.html).
+Though the practices listed in the introduction can be considered general enough to be applicable to any kind of software, individual communities usually add their own specific sets of rules and conventions that help users quickly find their way around software projects, understand them more easily and reuse them more effectively. The Galaxy community, for instance, has a [guide on best practices for maintaining workflows](https://planemo.readthedocs.io/en/latest/best_practices_workflows.html). These best practices include:
+
+* all workflow inputs and outputs are explicit and labelled
```

Comment:
this section has some overlap with https://training.galaxyproject.org/training-material/faqs/galaxy/workflows_best_practices.html perhaps it could make sense to add some of these details there and include the snippet here? `{% snippet faqs/galaxy/workflows_best_practices.nd %}`

Reply:
Thank you! I did wonder if this would have been mentioned elsewhere, but I didn't find it 😅 I've changed it so most of the detail is in the FAQ now